custom_components/tuya_local/devices/ekf_connect_pro_wifi_powerstrip.yaml

```diff
@@ -0,0 +1,153 @@
+name: EKF Connect Pro WiFi 4 outlet + USB powerstrip
```

Comment:
This one seems to be the same as the generic "quad_powerstrip.yaml" config, with the only addition being the non-useful test bit (which is always 1 in shipped devices)

Reply:
Should I remove the config for it then?